Apps/GatewayApi/test/unit/Services.Test/DependentServiceTests.cs

```diff
@@ -250,7 +250,7 @@ public void ValidateRemove()
             mockUserProfileDelegate.Setup(s => s.GetUserProfile(this.mockParentHdid))
                 .Returns(
                     new DbResult<UserProfile>
-                        { Payload = new UserProfile() });
+                    { Payload = new UserProfile() });
```

Comment:
Lines 253 and 416. You've changed the formatting.  I believe the team has tried to standardize formatting so that when files are saved and then checked, there aren't a lot of changes.

Reply:
fixed